### DISCUSSION TITLE
Linux Platform Support Contributors Group

### DISCUSSION THREAD

**Discussion by TatryProject** [zed-industries/zed#6936] [Feature Requests]
Hello! I understand Linux support is on the roadmap, however at what point will members of the community be able to contribute towards a release on the platform? As someone who doesn't use or own a Mac device, I would love to have a way to contribute toward development on other platforms. Further, I'm aware there are Fireside Hack sessions, though these appear to only be accessible through Zed, which isn't possible without a Mac. Is there a way for people on other platforms to engage in the community and receive updates, perhaps a Discord group or something? Thanks!

**Comment by tml** [comment]
Not intended as a direct answer to @aagacek2's question, as it seems they've already done their research here, but since others may come to this thread looking for this information: Discussion about platform support (Windows / Linux) can be found at https://github.com/zed-industries/zed/issues/5391

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
